examples/mongodbatlas_federated_settings_org_role_mapping/Readme.md

```diff
@@ -7,8 +7,8 @@ This project aims to provide an example of using Okta and MongoDB Atlas together
 
 * Terraform v0.13
 * Okta account 
-* A MongoDB Atlas account 
-
+* A MongoDB Atlas account with an Organization configured with [Federation Authentication](https://www.mongodb.com/docs/atlas/security/federated-authentication/#federation-management-console)
+  * Get the `federated_settings_id` from the url, e.g., <https://cloud.mongodb.com/v2#/federation/{federated_settings_id}/overview>
```

Comment:
can also be retrieved through `mongodbatlas_organization` resource or data source https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/organization#argument-reference